tests/lib/managers/folders-test.js

```diff
@@ -276,4 +276,18 @@ describe('Folders', function() {
 		});
 	});
 
+	describe('getTrashedFolder()', function() {
+		it('should make GET request to get trashed folder when called', function() {
+			sandbox.stub(boxClientFake, 'defaultResponseHandler');
+			sandbox.mock(boxClientFake).expects('get').withArgs('/folders/' + FOLDER_ID + '/trash', testParamsWithQs);
+			folders.getTrashedFolder(FOLDER_ID, testQS);
+		});
+
+		it('should call BoxClient defaultResponseHandler method with the callback when response is returned', function(done) {
```

Comment:
I was hoping you would comment about tests, because the approach with testing is not clear here. Could you please elaborate about the concept behinds tests. I was trying to mimic existing tests in this case it was test for get folder function:
```
	describe('get()', function() {
		it('should make GET request to get folder info when called', function() {
			sandbox.stub(boxClientFake, 'defaultResponseHandler');
			sandbox.mock(boxClientFake).expects('get').withArgs('/folders/1234', testParamsWithQs);
			folders.get(FOLDER_ID, testQS);
		});
```